lib/tasks/search.rake

```diff
@@ -12,6 +12,17 @@ namespace :search do
 
   desc "Recreate tag index"
   task(index_tags: :environment) do
+    if Rails.env.development? || Rails.env.test?
```

Comment:
Sorry, I had the wrong environment when I was talking with you earlier 🤦 
```suggestion
    if Rails.env.production? || Rails.env.test?
```

Reply:
Annoyed with myself for not noticing this myself!! Pushed the fix, thank you.